salt/orchestrate/aws/rds.sls

```diff
@@ -57,14 +61,35 @@ set_{{ name }}_master_password_in_vault:
 {% set master_pass = master_pass.data.value %}
 {% endif %}
 
+{# If engine is postgres and version is 10, then use major version only for
+   parameter group family. All others use major.minor. #}
+{% if engine == 'postgres' and int(engine_version.partition('.')[0]) > 9 %}
+{% set fam_version = engine_version.partition('.')[0] %}
+{% else %}
+{% set fam_version = engine_version.rpartition('.')[0] %}
+{% endif %}
+{% set db_parameter_group_family = engine ~ fam_version %}
+{% set db_parameter_group_name = ENVIRONMENT ~ '_' ~ engine ~ '_' ~ name ~ '_parameters' %}
```

Comment:
For the case where we are upgrading an RDS instance we might want to use the parameter group family in the name of the group, replacing `engine` with `db_parameter_group_family`.

Reply:
I've changed this in ace521b.